rustls/src/lib.rs

```diff
@@ -266,6 +266,10 @@
 //!
 //! [`mio`]: https://docs.rs/mio/latest/mio/
 //!
+//! # Detailed manual
```

Comment:
Why "detailed"? It seems a bit superfluous to me. Why not just refer to it as the manual?